gcloud/rest/kms/client.py

```diff
@@ -0,0 +1,54 @@
+import threading
+
+import requests
+
+from gcloud.rest.auth import Token
+
+
+API_ROOT = 'https://cloudkms.googleapis.com/v1'
+LOCATION = 'global'
+SCOPES = [
+    'https://www.googleapis.com/auth/cloudkms',
+]
+
+
+class KMS(object):
```

Comment:
Is there a vector for the users of `KMS` to get the keys that they've encrypted or is this purely for the use of the encryption and decryption service, not storage of encrypted keys?

Reply:
This is encryption/decryption only. At some point, it would be great to expand to all the other usages of this API, but for a v1 I'm just implementing the barebones.